ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/geb/AbstractModelCatalogueGebSpec.groovy

```diff
@@ -70,6 +75,27 @@ abstract class AbstractModelCatalogueGebSpec extends GebReportingSpec {
         return new DataModelNavigator(this)
     }
 
+    DataModelNavigator selectLatestVersion(String dataModelId, String dataModelName) {
```

Comment:
I would prefer updating the original method with some parameters such as `latest: true` or `id: dataModelId`